www/googlemaps-cdv-plugin.js

```diff
@@ -2632,10 +2632,13 @@ function getAllChildren(root) {
     var clickable;
     var style, displayCSS, opacityCSS, visibilityCSS, node, clickableSize;
 
-    var allClickableElements = Array.prototype.slice.call(root.querySelectorAll(':not([data-clickable="false"])'));
-    var clickableElements =  allClickableElements.filter(function(i) {return i != root;});
+    root = document.querySelector('.main-region.foreground');
```

Comment:
Yep, it is, but... to be honest, I don't know how this was working before, we are setting the map div to a sibling of the main region, which... shouldn't being iterated by this... 

This is just a workaround to avoid overcomplicating things in the plugin.